### ISSUE TITLE
[Issue]: User with max sessions can't log in even after all session are logged out without a restart

### ISSUE THREAD

**Issue by ferferga** [jellyfin/jellyfin#7140]
### Please describe your bug

When an user has a max session limit, user can't log in anymore, even if all sessions are logged out.

**Steps to reproduce**
* Set an user to have a maximum number of 1 simultaneous session
* Log in with that user in one device
* Log in with that user in another device. You will be greeted with an UserNoAuthorized error.
* (As an admin) Close all the sessions of that user from the Devices tab
* Try to login again with that user. It's impossible.
* Restart the server and attempt logging in again. It will work.

**Expected behaviour**
User can log in as soon as the session is closed by the admin in the devices tab.

### Jellyfin Version

10.7.7

### Environment

```markdown
- OS: Debian 11
- Virtualization: No
- Clients: Any
- Browser: Any
- FFmpeg Version: -
- Playback Method: -
- Hardware Acceleration: -
- Plugins: -
- Reverse Proxy: Nginx
- Base URL: -
- Networking: LAN
- Storage: HDD
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by LewkyB** [comment]
I attempted to reproduce this on `release-10.8.z` by using different browsers and I was able to log in normally each time without restarting.

**Comment by KaXaSA** [comment]
I had this issue yesterday on 10.8.7 
I also had the maximum number of simultaneous users per session set to 1

```
[2022-11-02 00:11:22.148 -03:00] [INF] [7] Jellyfin.Api.Auth.CustomAuthenticationHandler: "CustomAuthentication" was not authenticated. Failure message: "Invalid token."
[2022-11-02 00:11:22.150 -03:00] [INF] [26] Jellyfin.Api.Auth.CustomAuthenticationHandler: "CustomAuthentication" was not authenticated. Failure message: "Invalid token."
[2022-11-02 00:11:22.151 -03:00] [INF] [23] Jellyfin.Api.Auth.CustomAuthenticationHandler: "CustomAuthentication" was not authenticated. Failure message: "Invalid token."
[2022-11-02 00:11:22.152 -03:00] [INF] [9] Jellyfin.Api.Auth.CustomAuthenticationHandler: "CustomAuthentication" was not authenticated. Failure message: "Invalid token."
```

After I change it to 0 (disabled), restarted the server, then it got back to normal.

This may be related to #8140

**Comment by egjok3r** [comment]
I am having what I believe is the same issue. 
I'm part of a server which limits concurrent user logins, I have logged in via the Iphone app, I have then logged out and tried logging into the same server with a web browser and the firetv app. Both times I have received errors and have been unable to log in, when I try with the iphone app I can log in again. 

I have repeated this by downloaded and setting this up on my own pc and using my phone and firestick to login when accounts are limited. I either need to refresh the browser with the panel, delete devices from the log or change the account device limit.

**Comment by cvium** [comment]
Seems like a client issue. I can reproduce it with Swiftfin. It does not seem to logout.

**Comment by codecivet** [comment]
Can reproduce this issue on 10.8.13.

**Comment by razor2611** [comment]
I've having the same problem in 10.9.11, one of my users using Swiftfin had the same problem. Tried to up the "Maximum number of simultaneous user sessions" to 5 from 3 and it still didin't work. The only option that worked was to set it to 0 to be disabled and I'm guessing if I've restarted the server it will work again but that's not an option.. Can't wait for the day we will have the Max simultaneous video streams option so I can disable this crap.
